design/ceph-nfs-ganesha.md

```diff
@@ -73,37 +74,18 @@ metadata:
   # created.
   namespace: rook-ceph
 spec:
-  store:
-    # The name of the filesystem or object store CRD that ganesha will export
-    name: myfs
-    # Either "file" for a filesystem or "object" for the object store
-    type: file
-
   # NFS client recovery storage settings
-  clientRecovery:
-    # RADOS pool where NFS client recovery data is stored.
-    # In this example the data pool for the "myfs" filesystem is used.
+  RADOS:
```

Comment:
I'm not too much of a fan of having this in all uppercase, `rados` would be better in my opinion in the CRD naming.

Reply:
I'm fine with renaming this. Will I need to change the ceph orchestrator code to match as well?